tests/qos/qos_sai_base.py

```diff
@@ -2107,12 +2126,26 @@ def egressLossyProfile(
 
         srcport = dutConfig["dutInterfaces"][dutConfig["testPorts"]["src_port_id"]]
 
+        is_lossy_queue_only = False
+
         if srcport in dualtor_ports_for_duts:
             queues = "0-1"
         else:
-            queues = "0-2"
+            if isMellanoxDevice(duthost):
+                profile_content = dut_asic.run_redis_cmd(
```

Comment:
@JibinBao Could we do similar to cable_length "0m" check instead of BUFFER_PG:4??

Reply:
@kperumalbfn , 
Done. Can you please review it again?